### PR TITLE
finished simple game-core 

### PR DESCRIPTION
**Pull request by pqoqubbw**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules/
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode
```
Comment (ekaterinaborodavka) [inline .gitignore:3]: add /dist

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 node_modules/
-.DS_Store
\ No newline at end of file
+.DS_Store
```
Comment (ekaterinaborodavka) [inline .gitignore:2]: add /dist

File: src/model/Game.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        this.updateCellEvent.trigger({ x, y, sign: this.players[this.currentPlayerIndex].sign });
+      }
+
+      const winnerId = this.strategy.checkWin(this.board);
+      const isDraw = this.strategy.checkDraw(this.board);
+
+      if (winnerId > 0 || isDraw) {
+        this.winnerId = this.currentPlayerIndex + 1;
+        this.isFinished = true;
+        if (isDraw) this.winEvent.trigger('no one');
```
Comment (all-bear) [inline src/model/Game.ts:44]: please check what is bad smell:

https://en.wikipedia.org/wiki/Code_smell
https://maximilianocontieri.com/series/code-smells

you have "magic string" string here https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings

File: src/utils/Event.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { IEventProps } from '../@types/types';
+
+class Event implements IEventProps {
```
Comment (all-bear) [inline src/utils/Event.ts:3]: This is not Event, Event is what you triggering

**Review by all-bear — CHANGES_REQUESTED** [review]

**Pull request merged**